nym-vpn-core/crates/nym-vpn-api-client/src/client.rs

```diff
@@ -574,14 +620,17 @@ impl VpnApiClient {
 mod tests {
     use nym_crypto::asymmetric::ed25519;
 
+    use crate::response::NymVpnAccountStatusResponse;
+
     use super::*;
 
     const BASE_URL: &str = "https://nymvpn.com/api";
     const BASE_URL_PREVIEW: &str =
         "https://nym-dot-com-git-deploy-canary-nyx-network-staging.vercel.app/api";
 
     fn get_mnemonic() -> bip39::Mnemonic {
-        let mnemonic = "kiwi ketchup mix canvas curve ribbon congress method feel frozen act annual aunt comfort side joy mesh palace tennis cannon orange name tortoise piece";
+        let mnemonic = "pill element gift second woman among bronze word boost expire lady believe plastic boil athlete advice place limb unable canvas twin tower feel carpet";
```

Comment:
Is it ok to keep this mnemonic here?

Reply:
It's temporary mnemonics for a preview deployment that doesn't exist anymore. But I'll remove because it's useless anyway